README.md

```diff
@@ -17,3 +28,78 @@ Fossil solves the following use cases:
 - Local Time Series Database that stores data in a local DB file
 - Collect data over a TCP connection in fire and forget mode
 - Query data over a TCP connection
+
+## How to
+
+### Install
+```shell
+go install github.com/dburkart/fossil
+```
+or
+```shell
+git clone https://github.com/dburkart/fossil.git
+cd fossil
+go run main.go
+```
+
+### Usage
+```shell
+> fossil server -h
+Database for collecting and querying metrics
+
+Usage:
+  fossil server [flags]
+
+Flags:
+  -d, --database string   Path to store database files (default "./")
+  -h, --help              help for server
+  -p, --port int          Database server port for data collection (default 8001)
+      --prom-http int     Set the port for /metrics is bound to (default 2112)
+
+Global Flags:
+  -c, --config string   Path to the fossil config file (default "./config.toml")
+  -H, --host string     Host to send the messages (default "fossil://local/default")
+      --local           Configures the logger to print readable logs (default true)
+  -v, --verbose count   -v for debug logs (-vv for trace)
+```
+
+### Config
+```toml
+[fossil]
+port = 8000
+prom-port = 2112
+
+host = "fossil://local/default"
+local = true
+verbose = 2
+
+[database]
+directory = "data/default"
```

Comment:
I think we will automatically create 1 level of directory for the user, but not two... may need to update the database creation code.

Reply:
Hmm ok, so for example, any directory that we put here will house the structure for that database. For a clear example lets take different mounts:

```toml
[database]
directory = "/mnt/d/data"
```
The database creation code would create a directory in the configured directory named `default` where it would place the following
```
/mnt/d/data/default/database/*
/mnt/d/data/default/wal.log
```
Another example for a named database `weather`

```toml
[database.weather]
directory = "/mnt/e/db_data"
```
resulting in
```
/mnt/e/db_data/weather/database/*
/mnt/e/db_data/weather/default/wal.log
```

Reply:
Yeah, that makes sense to me-- I was just wondering what happens if the parent directory (data in this case) doesn't exist?

Reply:
Hmm, I mean I am happy to say we make it, but I would probably look to how pgsql and mysql handle this.

Reply:
I understand now. `directory` will need to exist, we can add a flag similar to `mkdir -P` but I think for now we focus on what you described. This is similar to what pg expects.